src/System.Security.Permissions/src/System/Security/Permissions/ResourcePermissionBase.cs

```diff
@@ -6,6 +6,7 @@
 
 namespace System.Security.Permissions
 {
+    [Serializable]
```

Comment:
@ViktorHofer Saw your name on this. Was leaving Serializable off intentional? Adding it IS causing the serialization tests to fail because System.Type is not marked as Serializable. I added it because its listed as such. However EventLogPermission and PerformanceCounterPermission fail the test because of this.